Phonebook.Frontend/src/app/shared/dialogs/user-information/Incorrect-user-information.component.ts

```diff
@@ -28,6 +28,6 @@ export class IncorrectUserInformationComponent implements OnInit {
     );
   }
   public get onMyProfile(): boolean {
-    return this.currentUserId.toLowerCase() === this.data.Id.toLowerCase();
+    return this.currentUserId.toLowerCase() === this.data.Id.toLowerCase( );
```

Comment:
Looks wrong for me?

Reply:
just formatting @Zeroks77 can you find the setting which does this?

Reply:
🤷‍♂️ hab gerade nochmal formatiert und es is normal geworden

Reply:
"normal"  - mit leerzeichen?
"normal" - ohne leerzeichen?